jsons/Buildings.json

```diff
@@ -3290,5 +3290,192 @@
+            "Unavailable <if [Great Musician purchased] is constructed> <hidden from users>",
+            "Only available <in [Capital] cities>",
+            "Provides a [Great Musician purchased] in your first [99] cities for free <hidden from users>",
+            "Remove [Purchase Great Musician] [in this city] <hidden from users>",
+            "Piety Complete Building"
+        ]
+    },
+    {
+        "name": "Great Musician purchased",
+        "uniques": [
+            "Unbuildable", "Unsellable",
+            "Will not be displayed in Civilopedia",
+            "Destroyed when the city is captured",
+            "Provides a [Great Musician purchased] in your first [99] cities for free <hidden from users>"
+        ]
+    },
+    {
+        "name": "Purchase Great Merchant",
+        "uniques": [
+            "Unbuildable", "Unsellable",
+            "Will not be displayed in Civilopedia",
+            "Free [Great Merchant] appears",
+            "Only available <after adopting [Piety Complete]>",
+            "Unavailable <if [Great Merchant purchased] is constructed> <hidden from users>",
+            "Only available <in [Capital] cities>",
+            "Provides a [Great Merchant purchased] in your first [99] cities for free <hidden from users>",
+            "Remove [Purchase Great Merchant] [in this city] <hidden from users>",
+            "Piety Complete Building"
+        ]
+    },
+    {
+        "name": "Great Merchant purchased",
+        "uniques": [
+            "Unbuildable", "Unsellable",
+            "Will not be displayed in Civilopedia",
+            "Destroyed when the city is captured",
+            "Provides a [Great Merchant purchased] in your first [99] cities for free <hidden from users>"
+        ]
+    },
+    {
+        "name": "Purchase Great Engineer",
+        "uniques": [
+            "Unbuildable", "Unsellable",
+            "Will not be displayed in Civilopedia",
+            "Free [Great Engineer] appears",
+            "Only available <after adopting [Piety Complete]>",
+            "Unavailable <if [Great Engineer purchased] is constructed> <hidden from users>",
+            "Only available <in [Capital] cities>",
+            "Provides a [Great Engineer purchased] in your first [99] cities for free <hidden from users>",
+            "Remove [Purchase Great Engineer] [in this city] <hidden from users>",
+            "Piety Complete Building"
+        ]
+    },
+    {
+        "name": "Great Engineer purchased",
+        "uniques": [
+            "Unbuildable", "Unsellable",
+            "Will not be displayed in Civilopedia",
+            "Destroyed when the city is captured",
+            "Provides a [Great Engineer purchased] in your first [99] cities for free <hidden from users>"
+        ]
+    },
+    {
+        "name": "Purchase Great Prophet",
+        "uniques": [
+            "Unbuildable", "Unsellable",
+            "Will not be displayed in Civilopedia",
```

Comment:
I think the great prophet is not needed here? You can buy it without buildings anyway.

Reply:
I guess. Looking at it, looks like the case for Lekmod too